school.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'highfiveable'
+
+class School
+  attr_reader :age, :phase
+  attr_accessor :name
+
+  include HighFiveable
+
+  def initialize(options={})
+    @phase = 3
```

Comment:
looks like phase is a shared attribute but just has a different value. How can this be DRYd up?